src/setupTests.js

```diff
@@ -6,4 +6,4 @@ import '@testing-library/jest-dom';
 
 configure({ adapter: new Adapter() });
 
-jest.useFakeTimers().setSystemTime(new Date('2020-05-06'));
+jest.useFakeTimers().setSystemTime(new Date('2020-05-05T04:00:00.000Z'));
```

Comment:
Oh nice, is this the solution???

Reply:
Yeah, I think the lack if offset hours tz causes the issue.   This way we explicitly set the time including day.